helpers/buildtemplate.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const stampTemplate = require('./stampTemplate.js');
+const convertTagNameToClassName = require("./convertTagNameToClassName.js");
+
+/**
+ * Create the template custom element file
+ */
+module.exports.CreateElementTemplate = (elementName) => {
+    const rawTemplate = fs
+    .readFileSync(require.resolve("./../templates/singleElement.js"))
+    .toString();
```

Comment:
I am 100% being anal here, but the `.readFileSync(...)` and the `.toString()` should be tabbed up